### PR TITLE
raspberry pi updated packages

### PR DESCRIPTION
**Pull request by expeditioneer**
I've updated

- raspberrypi-sources
- raspberrypi-firmware
- raspberrypi-wifi-ucode

also added myself as co-maintainer I hope this is OK - if not i can remove me as maintainer.

Should and can old versions of packages also be dropped?

### CODE REVIEW

**Review by thesamesam — APPROVED** [review]
Fine with me on all counts, as long as tested it. Thank you!

File: sys-firmware/raspberrypi-wifi-ucode/metadata.xml [hunk 1]
```diff
@@ -5,6 +5,10 @@
 		<email>sam@gentoo.org</email>
 		<name>Sam James</name>
 	</maintainer>
+	<maintainer type="person" >
```
[the remark this answers is not in the document]
Comment (expeditioneer) [reply sys-firmware/raspberrypi-wifi-ucode/metadata.xml:8]: I've removed the whitespace

**Comment by expeditioneer** [comment]
Should I drop old `/sys-kernel/raspberrypi-sources` in a separate commit or are they still needed?
What about `sys-boot/raspberrypi-firmware` is _1.20211029_ still needed?

**Comment by thesamesam** [comment]
Nah, just stable latest raspberrypi-firmware (file a bug). As for sources, from https://github.com/raspberrypi/linux/branches, it looks like they're only supporting 5.10, 5.15 now, so we should only do those too, I think.

**Comment by expeditioneer** [comment]
Stabilization bug for firmware [#836845](https://bugs.gentoo.org/836845).

**Pull request closed without merging**